export function handleError(e: any): void {
    console.error(e);
}